src/pages/Context.mdx

```diff
@@ -119,3 +119,38 @@ const Example = React.memo(props => {
 
 There is also a lot of custom context/global state implementations, comparison
 for Concurrent mode [available there](https://github.com/dai-shi/will-this-react-global-state-work-in-concurrent-mode).
+
+### Don't create objects directly in props of Provider
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/76087934-72ecd900-5fb7-11ea-8c30-81a08a73d978.png)
![image](https://user-images.githubusercontent.com/14183168/76087991-8c8e2080-5fb7-11ea-9d13-8ddcf02beb34.png)


It's mentioned in the initial section, I suggest to change it instead.

Reply:
overlooked it, changed it in the section above